lib/browserstack-service.js

```diff
@@ -12,8 +12,14 @@ class BrowserstackService {
       auth: this.auth
     }, function(error, response, body) {
       if (!error && response.statusCode === 200) {
-        const { device, browserName, os, os_version, browser, browser_version } = capabilities;
-        const browserString = device ? `${device} ${browserName}` : `${os} ${os_version} ${browser} ${browser_version}`;
+        // These keys describe the browser the test was run on
+        const browserDesc = ['device', 'os', 'osVersion', 'os_version', 'browserName', 'browser', 'browserVersion', 'browser_version'];
+        const browserString = browserDesc.reduce((res, k) => {
```

Comment:
It looks like reduce is not necessary here. Let's do `browserDesc.filter((k) => capabilities[k]).join(' ');`